safe/impact_function/impact_function.py

```diff
@@ -966,6 +972,33 @@ def prepare(self):
 
             return PREPARE_SUCCESS, None
 
+    def output_number(self):
+        """Compute the number of outputs that the IF will produce.
+
+        You must run `prepare` first or it will return None.
+
+        :return: The number of output.
+        :rtype: int
+        """
+        if not self._is_ready:
+            return None
+        else:
+            return self._output_number()
+
+    def _output_number(self):
+        """Compute the number of outputs that the IF will produce.
+
+        Be careful when you call this function.Better to use `output_number()`.
+
+        :return: The number of output.
+        :rtype: int
+        """
+        if is_raster_layer(self.exposure):
+            if self.exposure.keywords.get('layer_mode') == 'continuous':
+                return 4
```

Comment:
Can you give comment what does 4 or 5 mean? Or where is it come from? Perhaps listing the output layers in the method's docstring is good for the other developers.